ERROR:daisy.client:Block downscale/57 with read ROI [0:3200, 2048:4096, 10240:10592] (3200, 2048, 352) and write ROI [0:3200, 2048:4096, 10240:10592] (3200, 2048, 352) failed in worker 2019
Traceback (most recent call last):
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/client.py", line 85, in acquire_block
    yield block
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/task.py", line 179, in _process_blocks
    self.process_function(block)
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 688, in <lambda>
    process_function=lambda b: downscale_block(
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 640, in downscale_block
    out_data = skimage.measure.block_reduce(in_data, factor, np.mean)
NameError: name 'skimage' is not defined
ERROR:daisy.client:Block downscale/72 with read ROI [3200:6400, 8192:10240, 2048:4096] (3200, 2048, 2048) and write ROI [3200:6400, 8192:10240, 2048:4096] (3200, 2048, 2048) failed in worker 2019
Traceback (most recent call last):
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/client.py", line 85, in acquire_block
    yield block
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/task.py", line 179, in _process_blocks
    self.process_function(block)
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 688, in <lambda>
    process_function=lambda b: downscale_block(
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 640, in downscale_block
    out_data = skimage.measure.block_reduce(in_data, factor, np.mean)
NameError: name 'skimage' is not defined
ERROR:daisy.client:Block downscale/74 with read ROI [9600:9700, 4096:6144, 2048:4096] (100, 2048, 2048) and write ROI [9600:9700, 4096:6144, 2048:4096] (100, 2048, 2048) failed in worker 2019
Traceback (most recent call last):
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/client.py", line 85, in acquire_block
    yield block
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/task.py", line 179, in _process_blocks
    self.process_function(block)
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 688, in <lambda>
    process_function=lambda b: downscale_block(
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 640, in downscale_block
    out_data = skimage.measure.block_reduce(in_data, factor, np.mean)
NameError: name 'skimage' is not defined
